tests/plugin_integration/Events/WPML/WPML_Rewrite_Test.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Events\WPML;
+use Spatie\Snapshots\MatchesSnapshots;
+use TEC\Events\Custom_Tables\V1\Provider as CT1_Provider;
+use Tribe\Tests\Traits\With_Uopz;
+use Tribe\Events\Views\V2\Hooks;
+use Tribe__Events__Main as TEC;
+use Tribe__Date_Utils as Dates;
+use Tribe__Events__Venue as Venue;
+
+class WPML_Rewrite_Test extends \Codeception\TestCase\WPTestCase {
```

Comment:
Do not use the `WPTestCase` when using `WPLoader` in `loadOnly: true` mode as it will try to handle the database as the `WPDb` module is doing.
Look at the `ct1_migration` suite for an example. Extend the base Codeception `Test` class.